frontend/web/components/EditPermissions.tsx

```diff
@@ -1176,29 +1189,51 @@ const EditPermissions: FC<EditPermissionsType> = (props) => {
                                     {email}
                                   </div>
                                 </Flex>
-                                {role === 'ADMIN' ? (
-                                  <Flex className='table-column fs-small lh-sm'>
-                                    <Tooltip
-                                      title={'Organisation Administrator'}
-                                    >
-                                      {
-                                        'Organisation administrators have all permissions enabled.<br/>To change the role of this user, visit Organisation Settings.'
-                                      }
-                                    </Tooltip>
-                                  </Flex>
-                                ) : (
-                                  <Flex
-                                    onClick={onClick}
-                                    className='table-column fs-small lh-sm'
-                                  >
-                                    {matchingPermissions &&
-                                    matchingPermissions.admin
-                                      ? `${Format.camelCase(
-                                          level,
-                                        )} Administrator`
-                                      : 'Regular User'}
-                                  </Flex>
-                                )}
+                                <div className='d-flex  gap-2 table-column fs-small'>
+                                  {role === 'ADMIN' ? (
+                                    <div className=''>
+                                      <Tooltip
+                                        title={
+                                          <PermissionSummaryItem isAdmin />
+                                        }
+                                      >
+                                        {
+                                          'Organisation administrators have all permissions enabled.<br/>To change the role of this user, visit Organisation Settings.'
+                                        }
+                                      </Tooltip>
+                                    </div>
+                                  ) : (
+                                    <div onClick={onClick}>
+                                      {!matchingPermissions?.permissions
+                                        ?.length ? (
+                                        <Tooltip
+                                          title={
+                                            <div className='text-body gap-1 align-items-center d-flex'>
+                                              No Permissions{' '}
```

Comment:
Maybe we can be more explicit here and say something like 'No User-level Permissions' ? I'm conscious that the content in the tooltip (copied below) still sounds like we're suggesting that permissions do exist for the user, but then the table data suggests that they have no permissions. 

> Permissions for this user may exist at the Group or Role Level

Maybe we could change the tooltop content to

```
This user has no permissions assigned directly, but they may belong to a group that has permissions on this environment|project. 
```

Reply:
Yep agree, it's a confusing area without us having a permissions debugger so being more explicit makes sense.